Gemfile

```diff
@@ -1,72 +1,47 @@
 source 'https://rubygems.org'
 
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+# Rails stuff
 gem 'rails', '5.2.1'
-# The mysql2 gem is required to connect ActiveRecord to our database
+
+# Database stuff
 gem 'mysql2', '~> 0.5.2'
-# Use SCSS for stylesheets
-# Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
-# Use Uglifier as compressor for JavaScript assets
-gem 'uglifier', '>= 1.3.0'
-# Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '~> 4.2.2'
-# See https://github.com/rails/execjs#readme for more supported runtimes
-gem 'therubyracer', platforms: :ruby
 
-# Use jquery as the JavaScript library
+# Assets (CSS/JS) stuff
 gem 'jquery-rails'
-# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
+gem 'sass-rails', '~> 5.0'
 gem 'turbolinks'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '~> 2.0'
-# bundle exec rake doc:rails generates the API under doc/api.
-gem 'sdoc', '~> 1.0.0', group: :doc
-
-# Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use Unicorn as the app server
-# gem 'unicorn'
-
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
-
-group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug'
-end
-
-group :development do
-  # Access an IRB console on exception pages or by using <%= console %> in views
-  gem 'web-console', '~> 3.7'
-
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
-end
-
+gem 'uglifier', '>= 1.3.0'
 
-gem 'blacklight', '>= 6.10'
+# Blacklight stuff
+gem 'blacklight', '~> 6.10.1'
+gem 'blacklight-marc', '~> 6.0'
+gem 'blacklight-ris', git: 'https://github.com/upenn-libraries/blacklight-ris.git'
 gem 'blacklight_advanced_search', '>= 6.3'
-gem 'jettywrapper', '>= 2.0'
-gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw]
```

Comment:
This ones probably not a bad idea to keep but since I doubt anyone develops on windows/etc its fine to go too.

(can always add back if it we need it down the road)